storage/migrations/01_oasis_3_consensus.up.sql

```diff
@@ -36,8 +36,8 @@ CREATE TABLE oasis_3.transactions
 (
   block BIGINT NOT NULL REFERENCES oasis_3.blocks(height) DEFERRABLE INITIALLY DEFERRED,
 
-  txn_hash   HEX64 NOT NULL,
-  txn_index  UINT31,
+  txn_hash   HEX64,
+  txn_index  UINT31 NOT NULL,
```

Comment:
Nit: WDYT about moving this up and above the blank line, so we have
```
block
txn_index

txn_hash
nonce
...
```
to visually separate/highlight the PK?

Same in `events`, I'd slightly prefer to have txn_block and txn_index together, because they form a unit of sorts.

I checked our SQL statements and they're resilient to us reordering the columns.

Reply:
Sure, updated! 

Oops I just saw #235 , I went ahead and added your inline comments from there to this PR as well. Thanks for that